controllers/kmskey_controller.go

```diff
@@ -125,23 +125,19 @@ func (r *KMSKeyReconciler) SetupWithManager(mgr ctrl.Manager) error {
 func (r *KMSKeyReconciler) manageSuccess(ctx context.Context, log logr.Logger, kmskey *kmsiapi.KMSKey) error {
 	reason := kmsiapi.KMSKeyReasonIssued
 	msg := ""
-	log.Info("successfuly reconciled kms key")
+	log.Info("successfully reconciled kms key")
 	r.Recorder.Event(kmskey, core.EventTypeNormal, reason, msg)
```

Comment:
Nitpick: Should we make this consistent with line no 139, and either use `msg` or `message` consistently?